plate_mapper/plate_mapper.py

```diff
@@ -97,20 +101,50 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None):
             barseqs.append(line.split('\t'))
     barseq_f.close()
     print('  Done.')
+
+    # Read special sample definitions
+    specs = {}
+    if special_f:
+        print('Reading special sample definitions...')
+        next(special_f)  # skip header line
```

Comment:
It seems like this logic could be replaced with loading into a pandas dataframe? the benefit is that those parsers are really well used vetted